packages/framework/src/Model/AdvancedSearch/AdvancedSearchQueryBuilderExtender.php

```diff
@@ -1,22 +1,22 @@
 <?php
 
+declare(strict_types=1);
```

Comment:
out of curiosity, why are you declaring strict types here? There are no scalar typehints or return types in this file :slightly_smiling_face:

Reply:
oooh, the array, ok :smile: